forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -497,15 +493,18 @@ export default function CreateNewFilterDrawer({
       <List>
         <h5 style={{ fontWeight: "bold", fontSize: "18px" }}>
           <Translation>{(t) => t("Criteria")}</Translation>{" "}
-          <i title={t("This section is aimed to set the parameters used to filter the tasks")} className="fa fa-info-circle"></i>{" "}
+          <i
+            title={t(
+              "This section is aimed to set the parameters used to filter the tasks"
+            )}
+            className="fa fa-info-circle"
+          ></i>{" "}
         </h5>
         <div className="d-flex align-items-center mt-1">
           <input
             type="checkbox"
             checked={isMyTasksEnabled}
-            onChange={(e) =>
-              setIsMyTasksEnabled(e.target.checked)
-            }
+            onChange={(e) => setIsMyTasksEnabled(e.target.checked)}
```

Comment:
here we can just give the function inside onChange and expect the event there in side the function as parameter